jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -5,39 +5,41 @@
         <v-container>
           <v-row>
             <v-col>
-              <v-select
-                :items="link_types"
-                @change="link_type_selected"
+              <v-radio-group 
                 label="Link type"
                 hint="Type of linking to be done."
+                v-model="link_type"
                 persistent-hint
-              ></v-select>
+                row>
+                <v-radio
+                  v-for="item in link_types"
+                  :key="item"
+                  :label="item"
+                  :value="item"
+                ></v-radio>
+               </v-radio-group>
             </v-col>
           </v-row>
-        </v-container>
-        <v-container>
-          <v-row>
+          <v-row v-if="link_type == 'WCS'">

```

Comment:
If we do want to hide the fallback for now, we can just change this to `v-if="false"` and the backend will still default to the True case but always hide the UI element (so that it can easily be re-added later, if necessary).
```suggestion
          <v-row v-if="false">

```

Reply:
Yes, why don't we do this, then I'll merge.

This is way better than what I could have done. Thanks!